CODEOWNERS

```diff
@@ -7,7 +7,7 @@
 
 # Docs Site + Graph
 /docs/nx-cloud @StalkAltan @rarmatei @isaacplmann @juristr @bcabanes
-/docs/shared @bcabanes @isaacplmann @juristr
```

Comment:
This line needs to be moved up, above the /docs/nx-cloud line. CODEOWNERS is read top-down, and whatever is the last rule to match a file takes precedence, even if a more specific rule matched it first.